protos/schedules.proto

```diff
@@ -0,0 +1,29 @@
+syntax="proto3";
+
+package dataform.schedules;
+
+message Notification {
+  repeated string emails = 1;
+  bool success = 2;
+  bool failure = 3; 
+}
+
+message ScheduleOptions {
+  repeated string actions = 1;
+  bool include_dependencies = 2;
+  bool full_refresh = 3;
+} 
+
+message Schedule {
+  string name = 1; // required field
+  bool enabled = 2;
```

Comment:
suggest inverting this to `disabled`. that way, the default state for a text schedules.json file is to not state a value for this field.